modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -197,11 +197,13 @@ function render_modal_status(node, radioNet) {
 
 function format_wifirate(rate) {
 	var s = '%.1f\xa0%s, %d\xa0%s'.format(rate.rate / 1000, _('Mbit/s'), rate.mhz, _('MHz')),
-	    ht = rate.ht, vht = rate.vht,
-	    mhz = rate.mhz, nss = rate.nss,
-	    mcs = rate.mcs, sgi = rate.short_gi,
-	    he = rate.he, he_gi = rate.he_gi,
-	    he_dcm = rate.he_dcm;
+		ht = rate.ht, vht = rate.vht,
+		mhz = rate.mhz, nss = rate.nss,
+		mcs = rate.mcs, sgi = rate.short_gi,
+		he = rate.he, he_gi = rate.he_gi,
+		he_dcm = rate.he_dcm,
+		eht = rate?.eht ?? false, eht_gi = rate?.eht_gi ?? 0,
+		eht_dcm = rate?.eht_dcm ?? 0;
```

Comment:
This has one indentation too much.

Reply:
there were spaces that I replaced with one tab

Reply:
Hmm, stylistic 4 spaces after the tab? I'm indifferent, but some devs like those retained.

Reply:
Yeah, retain the tab + 4 spaces please

Reply:
The variables was by design left-aligned with the first line (`var s = ...`), so if there were spaces to keep that alignment those should be kept, it's a common practice employed in the code base.

Reply:
ok let me put the spaces back give me a couple of minutes

Reply:
done

Reply:
Looks good.

Reply:
No rush, we'll be here ;)